### DISCUSSION TITLE
Идеи

### DISCUSSION THREAD

**Discussion by ZiraiMode** [link1107/DonationExecutor#35] [Идеи и предложения]
Телепорт Игрока A к Игроку B (по никам)

Тотем неуязвимости

Выдача блока по id. Сделать несколько массивов со списком блока (обычные блоки, редкие, и так далее). Человек сможет выдавать блок из какой то категории. Категории сделаны для упрощения донатинга, каждый блок прописать нереально. То же самое можно сделать с другими предметами. Например еда, разные сеты брони.

BTW. Вынеси функции-реализации донатов в разные фичи-классы. В одном из пр'ов тебе это уже делали. А то у тебя GOD object там вырисовывается

**Comment by ZiraiMode** [comment]
BTW. Закрывай пр после мержа

**Comment by ZiraiMode** [comment]
BTW. Ты можешь повесить [слушатель на файл](https://stackoverflow.com/questions/494869/file-changed-listener-in-java) конфига и автоматически релоадить плагин при любых изменениях файла

**Comment by mirronake** [comment]
Вкину здесь свои идеи, 
Сделать в конфиге строчку, чтобы отображало ник донатера, если ТОТ авторизирован через Twitch в DA при донате, надеюсь API подобное позволяет.
Сделать авторизацию через твич, чтобы можно было за БАЛЛЫ КАНАЛА создавать ивенты. Типо ты подключаешь плагин к твичу и у чела появляются награды за баллы которые стример уже может настроить под свои нужды.

**Comment by ZiraiMode** [comment]
@link1107

**Comment by 4pOnk** [comment]
Бля, как тебе идея сделать случайное событие, и крутить рулетку у стримера, я хз, возможно ли это с уебанским api Donation Alerts, но, если получится, будет гипер ахуенно

**Comment by mkinPrivMesn** [comment]
Выложи этот плагин на спигот, так больше людей узнает о нём.

https://spigotmc.ru/

**Comment by twin333** [comment]
можете перенести плагин на один из этих сайтов что бы можно было добавить плагин на атернос
![image](https://user-images.githubusercontent.com/96664603/161486483-53eefc88-fa9e-401d-80c0-9abca03ab564.png)
